page-objects/lib/progressbar.po.ts

```diff
@@ -0,0 +1,48 @@
+import {BasePO} from '@agnos-ui/base-po';
+import type {Locator} from '@playwright/test';
+
+export const progressbarSelectors = {
+	rootComponent: '[role="progressbar"]',
+	outerBar: '.progress',
+	innerBar: '.progress-bar',
+};
+
+export interface State {
+	ariaLabel: string | null;
+	ariaValueNow: string | null;
+	ariaValueMin: string | null;
+	ariaValueMax: string | null;
+	ariaValueText: string | null;
+	label: string | null | undefined;
+	innerClasses: string[];
+}
+
+export class ProgressbarPO extends BasePO {
+	selectors = structuredClone(progressbarSelectors);
+
+	override getComponentSelector(): string {
+		return this.selectors.rootComponent;
+	}
+
+	get locatorOuterBar(): Locator {
+		return this.locatorRoot.locator(this.selectors.outerBar);
+	}
+	get locatorInnerBar(): Locator {
+		return this.locatorRoot.locator(this.selectors.innerBar);
+	}
+
+	async getState(): Promise<State> {
```

Comment:
I don't know if we must call `getState`or just `state` (as the rating). It's a decision to take, but we need to be consistent...

Reply:
sure, I will rename to state then

Reply:
note that I will not handle focustrack and transition pos that also currently use ```getState``` in this PR